frontend/app/src/helpers/helpers.ts

```diff
@@ -111,7 +111,7 @@ export const sendToRedirect = (url: string) => {
 
 export const calculateTimeLeft = (

```

Comment:
we should totally add some sort of unit test for this function

Reply:
we might need to do this in the future since we don't really have a testing framework in place rn